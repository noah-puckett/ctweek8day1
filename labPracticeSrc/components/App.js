import React from 'react';
import AddInput from '../containers/addInput';
import TitleSubtitleDisplay from '../containers/TitleSubtitleDisplay';

export default function App() {
    return (<>
    <AddInput />
    <TitleSubtitleDisplay />
    </>);
}
