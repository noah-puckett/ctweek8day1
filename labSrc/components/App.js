import React from 'react';
import {
    BrowserRouter as Router,
    Route,
    Switch
} from 'react-router-dom';
import TopWords from '../containers/TopWords';
import Searchable from './words/Searchable';

export default function App() {
    return (
        <Router>
            <Switch>
                <Route path="/words/:count" component={Searchable}/>
                <Route exact path="/" component={TopWords} />
            </Switch>
        </Router>
    );
}
