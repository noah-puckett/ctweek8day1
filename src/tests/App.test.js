import React from 'react';
import { shallow } from 'enzyme';
import App from '../components/App';

describe('App component', () => {
    it('renders App', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});

