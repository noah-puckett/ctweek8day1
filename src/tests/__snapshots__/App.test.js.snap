// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      path="/words/:count"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Words)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
  </Switch>
</BrowserRouter>
`;
